source/NVDAObjects/IAccessible/sysListView32.py

```diff
@@ -394,7 +394,11 @@ def _get_name(self):
 				textList.append("%s: %s" % (header, content))
 			else:
 				textList.append(content)
-		return "; ".join(textList)
+		name = "; ".join(textList)
+		#Some list view items in Vista and later can contain annoying left-to-right and right-to-left indicator characters which really should not be there.
+		name=name.replace(u'\u200E','')

```

Comment:
Wouldn't it be better to just return the modified name in one line like return name.replace(blah, "").replace(blah,"")?